packages/df-blogs/src/ViewPost.tsx

```diff
@@ -65,6 +69,24 @@ function ViewPostInternal (props: ViewPostProps) {
     </>;
   };
 
+  const renderCommentPreviews = () => {
+    if (!commentIds || commentIds.length === 0) {
+      return <em>This blog has no posts yet</em>;
+    }
+
+    return commentIds.map((id, i) => <ViewComment key={i} id={id} preview />);
+  };
+
+  const commentsSectionTitle = () => {
+    return <>
+      <span style={{ marginRight: '.5rem' }}>Comments ({commentsCount})</span>
+      <Link to={`/blogs/${id}/newComment`} className='ui tiny button'>
```

Comment:
See the comment about Route for this URL in `index.tsx` file.
```suggestion
      <Link to={`/blogs/posts/${id}/newComment`} className='ui tiny button'>
```